### ISSUE TITLE
Xiaomi Mi 11 support (found control file)

### ISSUE THREAD

**Issue by J3is** [sriharshaarangi/BatteryChargeLimit#210]
TL;DR control file for Mi 11 is `/sys/class/qcom-battery/input_suspend, 0, 1`.

Hello, I spent the day trying to make the app work on my new Xiaomi Mi 11.
I couldn't find a working control file inside `/sys/class/power_supply`, so I initially thought suspending charging wasn't supported on this model.
In disbelief I kept on searching, and I found that the control file is situated in `/sys/class/qcom-battery/input_suspend`. The values obviously are 0 for enabled and 1 for disabled, since it's the usual Xiaomi input_suspend, just moved to a different location.
I don't know if this is a Snapdragon 888 thing or just a Mi 11 oddity, however if there's anyone else having problems with this app on Snapdragon 888, you could try looking in this path.

Could you update the app to reflect this? I know it's not mandatory since you can configure it yourself, but I ask just for completeness' sake. Thank you and thanks for your work on this awesome app.

**Comment by SwiTool** [comment]
Thanks, as a Mi 11 user, I never searched the control file and you just saved me some precious time here!

**Comment by 0czko** [comment]
11t pro user here, miui 13/a12. Above fix works!

**Comment by 5ulo** [comment]
`/sys/class/qcom-battery/input_suspend, 0, 1` applies for Xiaomi 13 too

**Comment by stefan2030** [comment]
> `/sys/class/qcom-battery/input_suspend, 0, 1` applies for Xiaomi 13 too

Also applies top Poco F5
